pkg/envoy/ads/response_benchmark_test.go

```diff
@@ -153,7 +153,7 @@ func setupTestServer(b *testing.B) {
 		b.Fatalf("Failed to create service: %v", err)
 	}
 
-	proxy = envoy.NewProxy(envoy.KindSidecar, proxyUUID, proxySvcAccount.ToServiceIdentity(), nil)
+	proxy = envoy.NewProxy(envoy.KindSidecar, proxyUUID, proxySvcAccount.ToServiceIdentity(), nil, 1)
```

Comment:
It might be nice to surface any performance improvement we see in tests based on this change

Reply:
this test would actually have no differences compared to the last, since it's not doing any extra work here